tests/integration/inventory.networking

```diff
@@ -26,10 +26,6 @@ aws_apic_hostname=52.52.20.121
 aws_apic_username=ansible_github_ci
```

Comment:
Should we also remove the `aws_` variables from here if they are not needed anymore?